src/NServiceBus.Core/OpenTelemetry/Tracing/ActivityFactory.cs

```diff
@@ -27,7 +27,10 @@ public Activity StartIncomingActivity(MessageContext context)
         }
         else if (context.Headers.TryGetValue(Headers.DiagnosticsTraceParent, out var sendSpanId) && ActivityContext.TryParse(sendSpanId, null, out var sendSpanContext)) // otherwise directly create child from logical send
         {
-            activity = ActivitySources.Main.CreateActivity(name: ActivityNames.IncomingMessageActivityName, ActivityKind.Consumer, sendSpanContext);
+            // TryParse doesn't have an overload that supports changing the isRemote setting yet
```

Comment:
Maybe decide if the comments can be prefixed with `//TODO:` to make it easier to find in the future.

Reply:
I think we're trying to avoid committing TODO comments. That's why I have an open task to raise dedicated issues in the repo instead to address this.